src/Generators/Xliff.php

```diff
@@ -2,11 +2,14 @@
 
 namespace Gettext\Generators;
 
+use Gettext\Translation;
 use Gettext\Translations;
 use DOMDocument;
 
 class Xliff extends Generator implements GeneratorInterface
 {
+    const UNIT_ID_REGEXP = '/XLIFF_UNIT_ID: (.*)/';
```

Comment:
Maybe we could make this regexp more strict, like `'/$XLIFF_UNIT_ID: (.*)^/'`?

Reply:
I think you've got those backwards -- but I've added beginning and end anchors, thanks!

Reply:
Opps, yes, you're right 🤦‍♂ 
Thanks!